addon/lib/Feature.jsm

```diff
@@ -163,12 +168,17 @@ class Feature {
     // Note: This listener can't be added until after the treatment has been applied,
     // since we are initializing built-in TP based on the treatment.
     this.addBuiltInTrackingProtectionListeners();
+
+    await this.initBehaviorSummary();
   }
 
   addContentMessageListeners() {
     // content listener
     Services.mm.addMessageListener("TrackingStudy:InitialContent", this);
+    Services.mm.addMessageListener("TrackingStudy::NewTabOpenTime", this);
```

Comment:
Why does one have `TrackingStudy:*` and the other has `TrackingStudy::*` (one-vs-two `::`)?

Reply:
That's how you know which listeners Kamyar added and which listeners I added. :)

Reply:
in that case, can I propose something like single colons and we append your names to the events? Something like `"TrackingStudy:IntialContent:Bianca"` and `"TrackingStudy:NewTabOpen:Kamyar"`?